tests/test_browser_model.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from foxpuppet import FoxPuppet
+
+
+class TestBrowserModel(object):
+
+    def test_new_private_window(self, selenium):
+        """Tests opening a new private browsing window via menu"""
+        foxpuppet = FoxPuppet(selenium)
+        foxpuppet.browser.open_window(private=True)
+        assert len(foxpuppet.windows.all) == 2
+        assert foxpuppet.browser.is_private is False
+        foxpuppet.windows.focus(foxpuppet.windows.all[1])
```

Comment:
with `open_window` returning the handle (ultimately we might have it return a window object) we can just use this value to switch to it using Selenium.